pyeudiw/satosa/backend.py

```diff
@@ -173,17 +172,37 @@ def entity_configuration_endpoint(self, context, *args):
 
     def pre_request_endpoint(self, context, internal_request, **kwargs):
 
+        session_id = str(context.state["SESSION_ID"])
+        state = str(uuid.uuid4())
+
+        # Init session
+        try:
+            self.db_engine.init_session(
+                state=state,
+                session_id=session_id
+            )
+        except Exception as e:
+            self._log(context, level='error', message=str(e))
```

Comment:
please add some explanatory text that wraps the str(e)

Reply:
Resolved in [f074214](https://github.com/italia/eudi-wallet-it-python/pull/92/commits/f074214da63a742ebf802cfb71047e5a155ebb0e)